sql/src/test/java/org/apache/druid/sql/calcite/CalciteReplaceDmlTest.java

```diff
@@ -440,9 +440,8 @@ public void testReplaceWithoutCompleteOverwriteClause()
   {
     testIngestionQuery()
         .sql("REPLACE INTO dst OVERWRITE SELECT * FROM foo PARTITIONED BY ALL TIME")
-        .expectValidationError(invalidSqlIs(
-            "Missing time chunk information in OVERWRITE clause for REPLACE. "
-            + "Use OVERWRITE WHERE <__time based condition> or OVERWRITE ALL to overwrite the entire table."
+        .expectValidationError(invalidSqlContains(
+            "Incorrect syntax near the keyword 'OVERWRITE' at line 1, column 18."
```

Comment:
We do get the suggestion too in the error message, right?

Reply:
These messages come after the sql has been validated at https://github.com/apache/druid/blob/28914bbab82480f33a345913a30c32321d36fc37/sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidPlanner.java#L146.  Here the SQL cannot be validated as the acceptable syntaxes are OVERWRITE ALL/OVERWRITE WHERE where as the query has only OVERWRITE. This fails planning because of bad syntax